salt/_modules/metalk8s_checks.py

```diff
@@ -434,6 +442,33 @@ def route_exists(destination, raises=True):
     return error or True
 
 
+def containerd_filesystem(raises=True, **kwargs):
```

Comment:
```suggestion
def containerd_filesystem(raises=True, **_kwargs):
```
To have the argument but ignored